app/views/recipes/public_recipes.html.erb

```diff
@@ -0,0 +1,11 @@
+<section class="public">
+    <h1>Public Recipes</h1>
+    <ul>
+        <% @public_recipes.includes([:recipe_foods]).each do |public_recipe| %>
+        <li>
+        <%= render partial: "public_recipe", locals: {recipe: public_recipe}%>
+        </li>
+        <% end %>
+    </ul>
+    
```

Comment:
- [ ] Please make sure to display recipes ordered by the newest on the public recipes page as requested in the project requirements. Currently, recipes are displayed ordered by the oldest.

![image](https://github.com/sylvester-wamaya/Capstone-Project-Recipe/assets/83803414/16bab6dd-e6e4-4e89-a7e8-9361a94cd2a4)